scripts/transfer/gold-to-john.js

```diff
@@ -13,15 +14,15 @@ async function run (runtimeEnv, deployer) {
   const goldOwner = deployer.accountsByName.get('alice');
 
   // execute asset transfer transaction
-  await executeTransaction(deployer, {
-    type: TransactionType.TransferAsset,
-    sign: SignType.SecretKey,
+  await deployer.executeTx([{
+    type: types.TransactionType.TransferAsset,
+    sign: types.SignType.SecretKey,
     fromAccount: goldOwner,
     toAccountAddr: john.addr,
     amount: 1,
     assetID: goldAsset.assetIndex,
     payFlags: {}
-  });
+  }]);
 
   await balanceOf(deployer, john.addr, goldAsset.assetIndex);
```

Comment:
You forgot update this line 😅

Reply:
It's there. You have commented on the old change.

Reply:
I see you only update `scripts/0-gold.js`.

Reply:
We discussed